nemo/collections/nlp/data/language_modeling/megatron/indexed_retrieval_dataset.py

```diff
@@ -430,7 +430,7 @@ def get_chunk(self, chunk_id, force_no_padding=False):
         """
         if isinstance(chunk_id, (int, np.int64, np.int32)):
             ptr = self._index.get_chunk_address(chunk_id)
-            if self._index.retrieval_db:
+            if self._index.retrieval_db and (not force_no_padding):
```

Comment:
Can you change name `force_no_padding` to be more indicative of function?  Maybe `force_no_cont_text`?

Reply:
changed the name to `force_no_cont_ids`